lib/hot/keys/vim.lua

```diff
@@ -0,0 +1,137 @@
+local hkng = require("hot.widget")
+
+
+for group_name, rule in pairs({
+  vim_motion={ color="#009F00", rule = {name="vim"} },
+  vim_command={color="#aFaF00", rule = {name="vim"} },
+  vim_command_insert={color="#cF4F40", rule = {name="vim"} },
+  vim_operator={color="#aF6F00", rule = {name="vim"} },
+}) do
+  hkng.group_rules[group_name] = rule
+end
+
+
+local vim_keys = {
+
+  vim_motion={{
+    modifiers = {},
+    keys = {
+      ['`']="goto mark",
+      ["#"..0+10+9]='"hard" BOL',
+      ['#20']="prev line",
+      w="next word",
+      e="end word",
+      t=". 'till",
+      ['[']=". misc",
+      [']']=". misc",
+      f=". find char",
+      [';']="repeat t/T/f/F",
+      ["'"]=". goto mk. BOL",
+      b="prev word",
+      n="next word",
+      [',']="reverse t/T/f/F",
+      ['/']=". find",
+    }
+  },{
+    modifiers = {"Shift",},
+    keys = {
+      ['`']="toggle case",
```

Comment:
Is that specific to your keyboard layout?  (Vim uses `~`, which is `AltGr++` on a German layout)